tests/unit_test.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash -e
 # This is for use as a Github CI Pull Request Unit Test.
-echo "ALL_CHANGED FILES: ${ALL_CHANGED_FILES}."
-echo "CHANGED_PACKAGES: ${CHANGED_PACKAGES}."
+set -e
```

Comment:
This is already set on the shebang line.

Reply:
we aren't using the shebang though, as we're running it via directly calling `/usr/local/bin/bash`.